### DISCUSSION TITLE
srt-file-transmit: How do I perform a file transfer between instances?

### DISCUSSION THREAD

**Discussion by catshow** [Haivision/srt#1991] [Q&A] [answered]
Hi All,
I am trying to do a file transfer between AWS regions, and I can't seem to make anything happen.  I have UDP Port 5001 open on both sides.  What am I doing incorrectly.

Caller side:

srt-file-transmit file:///home/ubuntu/output.dat srt://<server-ip>:5001 -v -ll debug -r:1000

Listener side:

srt-file-transmit srt://:5001 file:///home/ubuntu/output.dat -v

**Comment by maxsharabayko** [comment, marked as answer]
On the caller side, you should specify the remote IP to connect to: `srt://ip:5001`

**Comment by re2zero** [comment]
Listener side:
srt-file-transmit srt://:5001/?mode=listener file:///home/ubuntu/ -v

**Reply by maxsharabayko** [reply]
Thank you @re2zero. There is indeed no need for the listener-receiver to provide the name of the file, as the caller-sender would share it in the handshake.

It is also worth noting that `srt://:5001` is implicitly treated as the listener, because no IP is provided to connect ("call") to.
Putting this reference here for interested readers: [srt-live-transmit.md#medium-srt](https://github.com/Haivision/srt/blob/v1.4.4/docs/apps/srt-live-transmit.md#medium-srt)
